packages/dev/docs/pages/react-aria/hooks.mdx

```diff
@@ -0,0 +1,167 @@
+{/* Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+import styles from '@react-spectrum/docs/src/docs.css';
+
+---
+category: Introduction
+type: hook
+navigationTitle: Getting Started
+---
+
+# React Aria Hooks
+
+This page describes how to get started with React Aria hooks.
+
+## Installation
+
+React Aria can be installed using a package manager like [npm](https://docs.npmjs.com/cli/npm) or
+[yarn](https://classic.yarnpkg.com/lang/en/).
+
+```
+yarn add react-aria
+```
+
+If you prefer, you can also use our hooks from individually versioned packages. This allows you to only
+install the hooks you use, or more granularly manage their versions. The individual packages are published
+under the [@react-aria](https://www.npmjs.com/org/react-aria) scope on npm. For example:
+
+```
+yarn add @react-aria/button
+```
+
+Once installed, hooks can be used from the monopackage or individual packages the same way.
+
+```tsx
+// Monopackage
+import {useButton} from 'react-aria';
+```
+
+```tsx keepIndividualImports
+// Individual packages
+import {useButton} from '@react-aria/button';
+```
+
+## Building a component
+
+React Aria provides behavior and accessibility through React Hooks. Since it does not provide any rendering,
+you are responsible for defining the DOM structure for your component and passing the DOM props returned by each
+React Aria hook to the appropriate elements. This is powerful because it allows you to be in complete control over
+the DOM structure that you render. For example, you may need to add extra elements for styling or layout control.
+You also get complete control over how you style your components: you could use CSS classes, inline styles, CSS-in-JS, etc.
+
+Start by importing the hook you wish to use, and calling it in your component. You'll typically pass through the props
+from your component, along with a [ref](https://reactjs.org/docs/refs-and-the-dom.html) to the DOM node in some cases.
+The hook will return one or more sets of DOM props which you should pass through to the appropriate element. This can be
+done by [spreading the props](https://reactjs.org/docs/jsx-in-depth.html#spread-attributes) returned from the hook onto
```

Comment:
```suggestion
from your component, along with a [ref](https://react.dev/learn/referencing-values-with-refs#refs-and-the-dom) to the DOM node in some cases.
The hook will return one or more sets of DOM props which you should pass through to the appropriate element. This can be
done by [spreading the props](https://react.dev/learn/passing-props-to-a-component#forwarding-props-with-the-jsx-spread-syntax) returned from the hook onto
```